gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -1021,39 +1021,37 @@ public Builder sourceOptions(Iterable<BlobSourceOption> options) {
        *
        * @return the builder
        */
-      public Builder target(BlobId target) {
-        this.target = BlobInfo.builder(target).build();
+      public Builder target(BlobId targetId) {
```

Comment:
Do you think we need to make it clear in Storage.copy command or CopyWriter class that copy copies the content AND information though latter could be overridden?